src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -57,6 +57,14 @@ const PageContent = ( {
 export default function WhatsNewModal( { showModal, onClose }: WhatsNewModalProps ) {
 	const locale = useI18nLocale();
 	const whatsNewPages: WhatsNewPage[] = [
+		{
+			image: selectiveSyncIllustration,
+			title: __( 'Choose what to sync' ),
+			description: __(
+				'Selective Sync allows you to sync only the files and folders you need, reducing the time and space required for your syncs.'
```

Comment:
Selective Sync is more of an internal code name than an official feature name. What about that direction?

```suggestion
				'Choose which files and folders to synchronize, speeding up the process and limiting changes to part of your site.'
```

Reply:
Thanks for the suggestion. I agree that selective sync is an internal name. Your version looks much better. I will update it as well.

Reply:
I like the second version but I feel like `part of your site` sounds a bit weird in there. Perhaps something like:

`Select only the files and folders you need to sync, streamlining the process and reducing unnecessary updates.`

Not a strong preference on my end though if you all already agreed on the wording.